Components/Button/button.html

```diff
@@ -37,7 +37,9 @@ <h1 class="header-heading">Vision UI</h1>
           <li class="sidebar-link active">
```

Comment:
Not sure why are you creating yourdoc site header again here in button component. But it seems as a bad practice to me.